spec/lib/ldap_authenticatable_tiny_spec.rb

```diff
@@ -107,42 +107,58 @@ def test_options(login, password)
 
   describe '#is_authorized_by_ldap? calls Net::LDAP correctly' do
     before { set_env }
+    let!(:pals) { TestPals.create! }
```

Comment:
In these tests we're now going to test against TestPals emails + default demo passwords instead of mocking everything.